src/common/InnerToolbar/InnerToolbar.js

```diff
@@ -35,6 +35,11 @@ const InnerToolbar = Radium(React.createClass({
         flex: 1,
         justifyContent: 'flex-end'
       },
+      tooltip: {
```

Comment:
instead remove this line: https://github.com/Syncano/syncano-dashboard/blob/devel/src/common/InnerToolbar/InnerToolbar.js#L68
please read documentations.